horreum-integration/src/test/java/io/hyperfoil/tools/HorreumTestExtension.java

```diff
@@ -319,4 +340,9 @@ public TestContainer(String composeFile, Map<String, String> envVariables) {
             withEnv(envVariables);
         }
     }
+
+    public static RequestSpecification bareRequest(){
```

Comment:
Where is this used? Integration tests should use the available clients directly, not through RestAssured.

Reply:
This is used in a subsequent PR. Will remove it from this one.